fedora_messaging/cli.py

```diff
@@ -155,3 +164,120 @@ def consume(exchange, queue_name, routing_key, callback, app_name):
                 str(e.reason),
             )
             sys.exit(e.exit_code)
+
+
+@cli.command()
+@click.option("--publish-exchange", help=_publish_exchange_help)
+@click.option("--file", help=_publish_file_help, required=True, type=click.File("r"))
+def publish(publish_exchange, file):
+    """
+    Send messages from provided file to an AMQP queue.
+    """
+    msgs_json_str = file.read()
+
+    try:
+        messages = loads(msgs_json_str)
+    except exceptions.ValidationError as e:
+        _log.error("Unable to validate message: %s", str(e))
+        sys.exit(1)
+    except KeyError as e:
+        _log.error("Unable to create message. Missing attribute %s", str(e))
+        sys.exit(1)
+    except ValueError as e:
+        _log.error("Unable to load serialized message: %s", str(e))
+        sys.exit(1)
+
+    for msg in messages:
+        _log.info("Sending message with topic %s", msg.topic)
+        try:
+            api.publish(msg, publish_exchange)
+        except exceptions.PublishReturned as e:
+            _log.error("Unable to publish message: %s", str(e.reason))
```

Comment:
Rather than exiting 1 here, I think it'd be good to use a different value than the default "something exploded" error status so scripts can potentially handle error cases differently. It's not a great mapping, but maybe https://docs.python.org/3/library/errno.html#errno.EREMOTEIO?